griptape/drivers/prompt/base_prompt_driver.py

```diff
@@ -68,6 +68,7 @@ def run(self, prompt_stack: PromptStack) -> TextArtifact:
                     for chunk in completion_chunks:
                         self.structure.publish_event(CompletionChunkEvent(token=chunk.value))
                         tokens.append(chunk.value)
+                    self.structure.publish_event(CompletionChunkEvent(token="\n"))
```

Comment:
Don't _love_ having to do this, but it's to resolve this issue:
![image](https://github.com/griptape-ai/griptape/assets/9911531/424a7ce3-77c8-4e60-a37d-7847a0499e91)

Somehow we need to signify that we've sent the final completion chunk. This solution seems relatively unoffensive since users could just trim this off if they don't need it.